separa.lista.js

```diff
@@ -1,25 +1,45 @@
 function categoria(cursos, categoria) {
-  
+  return cursos.filter((cursos) => {
+    if(cursos.categoria == categoria){
+      return cursos
+    }
+  })
 }
 
 function duracaoMaiorQue(cursos, duracao) {
-  
+  return cursos.filter((cursos) => {
```

Comment:
cursos.filter((**cursos**) -> aqui vc pode dar qualquer nome, mas cursos, no plural, me parece não estar coeso

Reply:
Além disso é uma má prática fazer um shadow de variáveis.

https://stackoverflow.com/questions/11901427/an-example-of-variable-shadowing-in-javascript